apps/remote_control/test/lexical/remote_control/code_intelligence/rename_test.exs

```diff
@@ -0,0 +1,319 @@
+defmodule Lexical.RemoteControl.CodeIntelligence.RenameTest do
+  alias Lexical.Document
+  alias Lexical.Project
+  alias Lexical.RemoteControl
+  alias Lexical.RemoteControl.CodeIntelligence.Rename
+  alias Lexical.RemoteControl.Search
+
+  alias Lexical.Test.CodeSigil
+  alias Lexical.Test.CursorSupport
+  alias Lexical.Test.Fixtures
+
+  import CodeSigil
+  import CursorSupport
+  import Lexical.Test.EventualAssertions
+  import Fixtures
+
+  use ExUnit.Case
+
+  setup_all do
+    project = project()
+    RemoteControl.set_project(project)
+    start_supervised!(Document.Store)
```

Comment:
This will be necessary to use `Document.Store.fetch(uri, :analysis)` as I suggested elsewhere.

```suggestion
    start_supervised!(Lexical.Server.Application.document_store_child_spec())
```

Reply:
good suggestion, but we can't just use that function, since here is in  `remote_control` app